### DISCUSSION TITLE
Take over oai maintainer role?

### DISCUSSION THREAD

**Discussion by sckott** [ropensci/oai#63] [General]
@mbojan I'm looking for a new job, and hence looking for new maintainers for my pkgs. Might you be interested in taking over the maintainer role?

**Comment by mbojan** [comment]
Oh! Does it mean you are effectively leaving rOpenSci? I can take over the maintenance, no problem.

**Reply by sckott** [reply]
Yep, leaving ropensci, not yet gone, but in the next few months or so.

I just made you admin on the repo. 

next version pushed to cran, i'll need to email them and say you're taking over

**Reply by mbojan** [reply]
OK, thanks. Good luck with finding a job!

**Reply by sckott** [reply]
thanks!
